examples/knative/powercli/kn-pcli-tag/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM photon:3.0
```

Comment:
General thought: we keep replicating this across all our examples, with +99% redundancy for the Dockerfile. How about creating our own PS SDK Docker image used across all examples?

Reply:
+1